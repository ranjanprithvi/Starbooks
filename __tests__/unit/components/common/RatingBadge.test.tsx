import "@testing-library/jest-dom";
import renderer from "react-test-renderer";
import RatingBadge from "../../../../src/components/common/RatingBadge";
import React from "react";

describe("RatingBadge", () => {
    it("renders correctly", () => {
        const tree = renderer.create(<RatingBadge rating={3} />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
