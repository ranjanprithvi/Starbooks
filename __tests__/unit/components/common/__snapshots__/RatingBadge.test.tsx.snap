// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RatingBadge renders correctly 1`] = `
<span
  className="chakra-badge css-4z4zi4"
>
  3
</span>
`;
